roles/solr_collection/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: python
+python: "2.7"
```

Comment:
This isn't the version of python we use, correct? Or was this autogenerated?

Reply:
This must have been copied from the example role Francis copied over from PUL! Should be removed since we're not using travis anymore, thanks for flagging